src/components/workspace-utils.js

```diff
@@ -139,9 +139,13 @@ export const SnapshotInfo = ({
         setSnapshotLoadError(undefined)
       } catch (e) {
         try {
-          setSnapshotLoadError(await e.json())
+          if (e.status === 403) {
+            setSnapshotLoadError('You do not have access to this snapshot.')
+          } else {
+            setSnapshotLoadError(`Unexpected error contacting Terra Data Repo: ${await e.json()}`)
+          }
```

Comment:
[Nit] I like the concision of

```suggestion
          e.status === 403 ?
            setSnapshotLoadError('You do not have access to this snapshot.') :
            setSnapshotLoadError(`Unexpected error contacting Terra Data Repo: ${await e.json()}`)          
```

I know it's very much a personal taste but I figured I'd put it out there as an alternative :)